C/services/common-plugin-interfaces/python/pyobject_reading_parser.cpp

```diff
@@ -108,13 +181,21 @@ std::vector<Reading *>* Py2C_getReadings(PyObject *polledData)
 			{
 				Reading* newReading = new PythonReading(polledData);
 				if (newReading)
+				{
+                    setReadingAttr(newReading, polledData, true);
+                    //Logger::getLogger()->info("Py2C_getReadings:L%d: reading=%s", __LINE__, newReading->toJSON().c_str());
+
+                    // Add the new reading to result vector
 					newReadings->push_back(newReading);
+				}
 			}
 		}
 	}
 	return newReadings;
 }
+#endif
 
+#if 0
```

Comment:
If we really don't need this code then let's remove it to avoid confusion.

Reply:
This file has been removed. Forgot to push it earlier.